test/rubygems/test_gem_stub_specification.rb

```diff
@@ -15,11 +15,11 @@ def setup
   end
 
   def test_initialize
-    assert_equal "foo", @foo.name
-    assert_equal Gem::Version.new("0.0.1"), @foo.version
+    assert_equal "foo",                        @foo.name
```

Comment:
Wouldn't be better instead of adding the alignment here to remove it elsewhere? Looking the very same diff side by side [here](https://github.com/rubygems/rubygems/pull/6209/files#diff-c41f72b806d91dc3bf46c805909ef4d091e63a35cbae5fd2349c3d6dafac50b7R36)the lines are wrapped just because of the spaces and the end result is less readable.

Anyway, style changes like this should be separate commit IMHO.

Reply:
I agree with both: I prefer the non aligned style, and I prefer not mixing in style changes 👍.

Reply:
I removed the style change from my commit.